packages/flutter_tools/lib/src/dart/analysis.dart

```diff
@@ -14,10 +14,11 @@ import '../base/utils.dart';
 import '../globals.dart';
 
 class AnalysisServer {
-  AnalysisServer(this.sdkPath, this.directories);
+  AnalysisServer(this.sdkPath, this.directories, { this.useCFE });
 
   final String sdkPath;
   final List<String> directories;
+  final bool useCFE;
```

Comment:
Nit: I think Dart's naming conventions say this should be `useCfe` (since the acronym has three or more letters).

Reply:
updated